overlord/devicestate/devicestate.go

```diff
@@ -543,6 +596,12 @@ func remodelEssentialSnapTasks(ctx context.Context, st *state.State, pathSI *pat
 
 	revOpts := revisionOptionsForRemodel(newModelSnapChannel, ms.newRequiredRevision, ms.newModelValidationSets)
 
+	newSnapID := ""
```

Comment:
We prefer the var notation for default values
```suggestion
	var newSnapID string
```